packages/blog-starter-kit/themes/hashnode/components/analytics.tsx

```diff
@@ -4,18 +4,20 @@ import { v4 as uuidv4 } from 'uuid';
 
 import { useAppContext } from './contexts/appContext';
 const GA_TRACKING_ID = 'G-72XG3F8LNJ'; // This is Hashnode's GA tracking ID
+const HASHNODE_ANALYTICS_URL = 'https://user-analytics.hashnode.com/api/analytics'; // This is Hashnode's internal analytics URL
 const isProd = process.env.NEXT_PUBLIC_MODE === 'production';
 const BASE_PATH = process.env.NEXT_PUBLIC_BASE_URL || '';
 
 export const Analytics = () => {
-	const { publication, post } = useAppContext();
+	const { publication, post, series, page } = useAppContext();
```

Comment:
A post that is part of a series should also send the series id. That is currently not happening.